src/applications/vaos/appointment-list/index.jsx

```diff
@@ -11,7 +11,11 @@ import CommunityCareAppointmentDetailsPage from './components/CommunityCareAppoi
 import ExpressCareDetailsPage from './components/ExpressCareDetailsPage';
 import useManualScrollRestoration from '../hooks/useManualScrollRestoration';
 
-function AppointmentListSection({ featureHomepageRefresh }) {
+function AppointmentListSection() {
+  const featureHomepageRefresh = useSelector(
+    state => selectFeatureHomepageRefresh(state),
+    shallowEquals,
```

Comment:
This selector returns a boolean, so this isn't necessary (and might be causing problems)